spec/support/shared/examples/msf/db_manager/session.rb

```diff
@@ -153,7 +161,7 @@
               it 'should not find workspace from session' do
                 db_manager.should_not_receive(:find_workspace)
 
-                report_session
+                expect { report_session }.to change(Mdm::Vuln, :count).by(1)
```

Comment:
It is not obvious from the test description, "should not find workspace from session", that this expect would be here.  Why is it here?